R/badger.R

```diff
@@ -71,8 +89,11 @@ create_badges <- function(badges_data, output_file_name = NULL,
                     rowwise() %>%
                     mutate(cardcode = make_participant(first, second, role, extra)))$cardcode,
                  collapse = "\n")
-  options(warn=0)
-  s <- glue::glue(DEFAULT_TEMPLATE)
+  options(warn = 0)
+
+  TEMPL <- read_template(template)
+  s <- glue::glue(TEMPL, .open = "<<", .close = ">>")
+  print(s)
```

Comment:
I think that this is left over from some testing you were doing, can we ditch it?